docs/user/TroubleShooting.md

```diff
@@ -0,0 +1,122 @@
+# Important
+When troubleshooting a **Master/Control Plane Machine**, it's **absolutely imperative** that you familiarize yourself with determining the health of the etcd members.  On some rare occasions, a Node may go unready on multiple master Machines, but one or more of those Machines may have healthy etcd members.  Before selecting a master Machine to delete, it's mandatory to determine that the Machine you are intended to delete will not compromise etcd quorum.
```

Comment:
I'm leaving the reference to "Masters" here because that matches current product docs in an effort to reduce confusion.  Once that is sorted up in the product docs, we'll remove it here as well.